### DISCUSSION TITLE
It would be great if we have Specs displayed in a Tree view under Test Explorer

### DISCUSSION THREAD

**Discussion by nagamanickamm** [getgauge/gauge-vscode#881] [Ideas]
Current Test Explorer shows

```
Feature1:
      Scenario 1
      Scenario 2
Feature1:
      Scenario 1
```

What is really good to have is a exact tree structure of this spec, example

```
Folder 1
     Feature1:
            Scenario 1
            Scenario 2
     Feature 2
            Scenario 1
Folder 2
      Feature 3 
         Scenario 1

```

**Comment by nagamanickamm** [comment]
Folder represents Business Area / Epic / System / Service

**Comment by sriv** [comment]
The top level nodes are the specifications, and the leaf nodes are the Scenarios. Gauge does not infer anything from folder structure. 

The API that lists the Specs does have a `filename` field, but IIRC it contains the filename and not the folder path. Need to check this.
